packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -28,6 +28,8 @@ const dev = env.NODE_ENV === 'development' || env.DEBUG_PROD === 'true';
 // Allows running tsh in insecure mode (development)
 const isInsecure = dev || argv.includes('--insecure');
 
+const PREHOG_ADDR = 'https://reporting-staging.teleportinfra.dev'; // TODO(gzdunek): change to prod before going live
```

Comment:
See, if we had a proper config file for dev & prod, we could use staging in dev and the prod address in the packaged app.

We have to spend a little bit of time on that some time. I'm thinking of adding something like [dotenv](https://github.com/motdotla/dotenv). Perhaps there are other libs which are better suited for the kind of app we're working on.

Reply:
Sure, I'm happy to do it in a proper way. I think for now we can leave it as it is (we don't have prod env for Connect yet) and I will fix it soon.